applications: nrf_desktop: Update USB state documentation (#9426)
nrfconnect/sdk-nrf: applications/nrf_desktop/doc/usb_state.rst

Low latency device configuration
================================

The module sets default value of :kconfig:option:`CONFIG_USB_HID_POLL_INTERVAL_MS` to ``1``.

> Requesting polling rate of 1 ms is done to ensure low latency of HID reports.

Reviewer: ```suggestion
For low latency of HID reports, the device requests a polling rate of 1 ms.
```
